lib/optimizely.rb

```diff
@@ -137,6 +137,9 @@ def get_variation(experiment_key, user_id, attributes = nil)
         }, @logger, Logger::ERROR
       )
 
+      experiment = @config.get_experiment_from_key(experiment_key)
```

Comment:
@rashidsp Can we remove this check from here now? 
https://github.com/optimizely/ruby-sdk/blob/master/lib/optimizely/decision_service.rb#L60